Automattic/jetpack
# Form block: fix preserving new line in form submission

Fixes #25007

#### Changes proposed in this Pull Request:

* Replacing `sanitize_text_field` with `sanitize_textarea_field` so new lined can be preserved on form submission.

#### Other information:


#### Jetpack product discussion



n/a

#### Does this pull request change what data or activity we track or use?



no

#### Testing instructions:






* Add contact form to a post and add multiple line content. 
* Submitted content will be in one line.
* Apply this branch and retry the procedure.
* New lines in content should be there.
